reconcile/utils/terrascript_aws_client.py

```diff
@@ -1663,12 +1667,11 @@ def populate_tf_resource_s3(self, spec):
                 "grant"
             )
         server_side_encryption_configuration = common_values.get(
-            "server_side_encryption_configuration"
+            "server_side_encryption_configuration", DEFAULT_S3_SSE_CONFIGURATION
```

Comment:
would you consider using
`common_values.get("server_side_encryption_configuration") or DEFAULT_S3_SSE_CONFIGURATION`

just in case someone would ever have a `server_side_encryption_configuration: null` in their s3 defaults file.

definitely a nit kind-of-thing though

Reply:
Done. Thank you.